### DISCUSSION TITLE
Why can't import project from my git organization ?

### DISCUSSION THREAD

**Discussion by bensbenj** [vercel/community#2146] [Help] [answered]
Vercel doesn't have access of repo from my organization, just from my personal repo..

someone can help me please ?

**Comment by amyegan** [comment, marked as answer]
Hi @bensbenj! If you are using a Hobby account, this is one of the [limitations](https://vercel.com/docs/concepts/limits/overview#connecting-a-project-to-a-git-repository). GitHub Organizations can only be connected to teams.

> ​Vercel does not support connecting your Personal Account's Projects to Git repositories owned by Git organizations. You can either switch to an existing Team or create a new one.
>
> The same limitation applies in the Project creation flow when importing an existing Git repository or when cloning a Vercel template to a new Git repository as part of your Git organization.
> —[Connecting a Project to a Git Repository](https://vercel.com/docs/concepts/limits/overview#connecting-a-project-to-a-git-repository)
